.ci/tav.json

```diff
@@ -4,6 +4,7 @@
   "modules": [
     "@apollo/server",
     "@aws-sdk/client-s3",
+    "@aws-sdk/client-dynamodb",
```

Comment:
Note for reviewer: still under the limit and I guess the matrix will be reduced once we drop older node versions.